packs/elastic/scripts/logsfeed.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
```

Comment:
Moving files it's ok no problem. But for the compose file I will it in 
its place, because the default naming is nice.

It's just for testing purpose for dev usage. When you test a pack and 
bringing up docker env using compose you just know right away into 
which container to login)

On Wed, May 20, 2015 at 6:47 PM, Tomaz Muraus 
<notifications@github.com> wrote:
> In packs/elastic/scripts/logsfeed.py:
> 
> > @@ -0,0 +1,99 @@
> > +#!/usr/bin/env python
> I'm slightly confused - what changes if we move it to etc/ directory?
> 
> In any case, not moving that file is not a blocker right now so if it 
> makes things easier, do leave it there.
> 
> On a related note - similar as other scripts, docker-compose.yaml is 
> also not really useful without docs. It would be great to add some 
> docs on how to use it to the readme.
> 
> —
> Reply to this email directly or view it on GitHub.
>